src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/SynchronizationTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.VisualStudio.Razor.IntegrationTests;
+
+public class SynchronizationTests : AbstractRazorEditorTest
+{
+    [IdeFact(Skip = "Not yet fixed")]
+    public async Task CSharpComponentBacking_UpdatesComponents()
```

Comment:
I was having a problem where there was instability that I was having trouble tracking down, but I was pretty sure I had a way to work around it. I didn't want to lose the repro for the issue since it may be a real problem, but also didn't want to have that instability blocking all our tests.

Solution: I created this class for which all tests are (as of right now) are known flaky).